IdokladSdk/Validation/ApiValidator.cs

```diff
@@ -49,6 +49,18 @@ private static IEnumerable<object> GetNestedObjects(object entity)
         private static bool CheckIfDateTimePropertiesAreUtc(object entity, IList<ValidationResult> validationResults)
         {
             var type = entity.GetType();
+            if (type == typeof(DateTime) || Nullable.GetUnderlyingType(type) == typeof(DateTime))
+            {
+                var dateTime = entity as DateTime?;
```

Comment:
Tady je potenciál pro refaktoring. Skoro stejný kód se používá pak v tom foreach cyklu níže. Dalo by se to vytáhnout do samostatné metody a použít. Nechám na tobě.